pkg/workload/cli/run.go

```diff
@@ -86,10 +87,17 @@ var individualOperationReceiverAddr = sharedFlags.String(
 var histograms = runFlags.String(
 	"histograms", "",
 	"File to write per-op incremental and cumulative histogram data.")
+
+var histogramExportFormat = runFlags.String(
+	"histogram-export-format", "json",
+	"Export format of the histogram data into the `histograms` file. Options: [ json, openmetrics ]")
 var histogramsMaxLatency = runFlags.Duration(
 	"histograms-max-latency", 100*time.Second,
 	"Expected maximum latency of running a query")
 
+var openmetricsLabels = runFlags.String("openmetrics-labels", "",
+	"Comma separated list of key value pairs used as labels, used by openmetrics exporter. Eg 'cloud=aws, workload=tpcc'")
```

Comment:
Could add here that both label key and value should be in open-metrics format, otherwise we would throw error.